src/swell/suites/process_obs/experiment.yaml

```
@@ -0,0 +1,74 @@
+# Experiment name and location
+experiment_id: x0044-jedi-process_obs
+experiment_root: /discover/nobackup/${USER}/JediExperiments/
```

```suggestion
experiment_root: /discover/nobackup/${USER}/SwellExperiments/
```
So it's consistent with the other experiments.